processes/ingest/loc.py

```diff
@@ -215,28 +216,26 @@ def addHasPartMapping(self, resultsRecord, record):
 
     def storePDFManifest(self, record):
```

Comment:
What are your thoughts on pulling this function out? I think this is duplicate code in many of the ingest processes.

Reply:
I can pull this function out and I'll refactor it in my next commit.

Reply:
ok sounds good!